src/main/java/io/confluent/connect/jdbc/source/JdbcSourceTask.java

```diff
@@ -151,21 +151,28 @@ public void start(Map<String, String> properties) {
       boolean mapNumerics
           = config.getBoolean(JdbcSourceTaskConfig.NUMERIC_PRECISION_MAPPING_CONFIG);
 
+      final TimestampHelper timestampHelper;
+      if (timestampColumns.size() == 1) {
```

Comment:
Empty list is allowed to be returned by the `ConfigDef` if the `String` representation of the list is empty. What should we do if users have supplied empty list of `timestampColumns`?`

Reply:
Yeah, that's a bit confusing. It is an option a parameter, but the helper is only used if the mode is timestamp or timestamp-incrementing. Are we missing a validator to make sure that at least one timestamp column is provided when the mode is timestamp or timestamp-incrementing?